### PR TITLE
Ddt 945 get shipment events itests

### PR DESCRIPTION
**Pull request by FinnAtAsseco**

### CODE REVIEW

File: src/test/java/bkg/v1/ShipmentEventIT.java [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    given()
+      .contentType("application/json")
+      .get("/v1/events")
+      .then()
+      .assertThat()
+      .statusCode(200)
+      .contentType(ContentType.JSON)
+      .header("API-Version", equalTo("1.0.0"))
+      .header("Current-Page", matchesRegex("^https?://.*/v1/events\\?cursor=[a-zA-Z\\d]*$"))
+      // .header("Next-Page", matchesRegex("^https?://.*/v1/events\\?cursor=[a-zA-Z\\d]*$"))
```
Comment (mp-assco) [inline src/test/java/bkg/v1/ShipmentEventIT.java:46]: why are these two commented out?

Comment (pimg) [inline src/test/java/bkg/v1/ShipmentEventIT.java:46]: remove commented out code

**Review by pimg — CHANGES_REQUESTED** [review]

File: src/test/java/bkg/v1/ShipmentEventIT.java (hunk 1 above)
Comment (FinnAtAsseco) [reply src/test/java/bkg/v1/ShipmentEventIT.java:46]: BKG does not send those headers

Comment (FinnAtAsseco) [reply src/test/java/bkg/v1/ShipmentEventIT.java]: Done

**Review by pimg — APPROVED** [review]

**Pull request merged**